pages/draft/2019-09/release-notes.md

```diff
@@ -1,172 +1,169 @@
----
-title: 2019-09 Release Notes
-section: docs
----
+### Draft 2019-09 Release Note
 
 For the vast majority of schema authors, we hope that these changes are minimally disruptive.
 
-The most likely to be frustrating is that `format` is no longer treated as a validation assertion _by default_ (although it is still possible for an application or user to configure a validator to treat it as one).  We decided this was acceptable because many schema authors are already extremely frustrated by its inconsistent behavior.
+The most likely to be frustrating is that `format` is no longer treated as a validation assertion *by default* (although it is still possible for an application or user to configure a validator to treat it as one). We decided this was acceptable because many schema authors are already extremely frustrated by its inconsistent behavior.
 
 For implementors, there is a lot more to consider, and further guidance on implementation topics will be forthcoming.
-* [Incompatible Changes](#incompatible-changes)
-* [Semi-incompatible Changes](#semi-incompatible-changes)
-* [Annotations, Errors, and Outputs](#annotations-errors-and-outputs)
-* [Keyword Changes](#keyword-changes)
-    * [Core Vocabulary](#core-vocabulary)
-    * [Applicator Vocabulary](#applicator-vocabulary)
-    * [Validation Vocabulary](#validation-vocabulary)
-    * [Format Vocabulary](#format-vocabulary)
-    * [Content Vocabulary](#content-vocabulary)
-    * [Meta-Data Vocabulary](#meta-data-vocabulary)
-    * [Hyper-Schema Vocabulary](#hyper-schema-vocabulary)
+
+- [Incompatible Changes](https://json-schema.org/draft/2019-09/release-notes#incompatible-changes)
+- [Semi-incompatible Changes](https://json-schema.org/draft/2019-09/release-notes#semi-incompatible-changes)
+- [Annotations, Errors, and Outputs](https://json-schema.org/draft/2019-09/release-notes#annotations-errors-and-outputs)
+- [Keyword Changes](https://json-schema.org/draft/2019-09/release-notes#keyword-changes)
+  - [Core Vocabulary](https://json-schema.org/draft/2019-09/release-notes#core-vocabulary)
+  - [Applicator Vocabulary](https://json-schema.org/draft/2019-09/release-notes#applicator-vocabulary)
+  - [Validation Vocabulary](https://json-schema.org/draft/2019-09/release-notes#validation-vocabulary)
+  - [Format Vocabulary](https://json-schema.org/draft/2019-09/release-notes#format-vocabulary)
+  - [Content Vocabulary](https://json-schema.org/draft/2019-09/release-notes#content-vocabulary)
+  - [Meta-Data Vocabulary](https://json-schema.org/draft/2019-09/release-notes#meta-data-vocabulary)
+  - [Hyper-Schema Vocabulary](https://json-schema.org/draft/2019-09/release-notes#hyper-schema-vocabulary)
```

Comment:
Why changing the links from relative to absolute?

Reply:
I don't understand what you mean?